ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleAdapter.java

```diff
@@ -10,4 +10,4 @@
  * Google Mobile Ads SDK mediation.
  */
 @Keep

```

Comment:
Admob suggest us using androidx Keep annotation and their latest sdk supports it. I think we should keep it as usual because we need to add more dependency and update our proguard and do regression testing. But in my opinion we can list it as a candidate 'support AndroidX' for our sdk goals in future.

Reply:
agree, we did some initial testing with Android X but havent fully wrapped up all the scenarios and impact on dependencies